src/lib.rs

```diff
@@ -170,8 +172,8 @@ impl ImGui {
             })
         }
     }
-    pub fn set_texture_id(&mut self, value: usize) {
-        self.fonts().set_texture_id(value);
+    pub fn set_texture_id(&mut self, value: ImTexture) {
```

Comment:
I think we should now rename this function to `set_font_texture_id`, since the font texture is not going to be the only texture anymore.